tools/IceRpc.Protobuf.Tools/OutputFileNamesTask.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) ZeroC, Inc.
+
+using IceRpc.CaseConverter.Internal;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace IceRpc.Protobuf.Tools;
+
+/// <summary>A MSBuild task to compute the C# file name generated from a given proto file.</summary>
+public class OutputFileNamesTask : Task
+{
+    /// <summary>Gets or sets the Protobuf source files.</summary>
+    [Required]
+    public ITaskItem[] Sources { get; set; } = Array.Empty<ITaskItem>();
+
+    /// <summary>Gets the Protobuf computed sources. The computed sources are equal to the <see cref="Sources" /> but
+    /// with the additional <c>OutputFileName</c> metadata.</summary>
```

Comment:
I would add a cref to the ComputedSource in the main ProtocTask.

Reply:
I would clean these coments in a follow-up PR